app/scripts/lib/mailcheck.js

```diff
@@ -69,14 +69,21 @@ function (Tooltip, Url) {
 
           tooltip.render();
 
-          tooltip.$el.on('click', 'span', function () {
-            element.val(suggestion.full);
-            // the user has used the suggestion
-            metrics.logEvent('tooltip.mailcheck-used');
-            tooltip._destroy();
+          tooltip.$el.on('click keypress', 'span', function (e) {
+            // if a click event is triggered or an enter key is pressed, destroy
+            // the tooltip.
+            if (e.type === 'click' || e.which === 13) {
+              element.val(suggestion.full);
+              // the user has used the suggestion
+              metrics.logEvent('tooltip.mailcheck-used');
+              tooltip._destroy();
+            }
           });
         }
       });
     }
+    if (element.length) {
```

Comment:
Instead could we `return` right after `var element = $(target);` if it cannot be found